README.md

```diff
@@ -21,9 +22,6 @@ In order for this script to check warranty status of the device, the device must
 - IBM script points to warranty info not related to the SKU, serial given
 - If a Meraki product has a licence with a renewal required state, the expiration date will be set to the current date
 
-## Change Log
```

Comment:
Why was this removed?

Reply:
I will add it back for future changes but at the time it was not being used